packages/app-degree-pages/src/components/ListingPage/components/ProgramList/ListView/index.colums.config.js

```diff
@@ -67,11 +67,18 @@ const columns = [
       const isOnline = resolver.isOnline();
       const directUrl = isOnline
         ? resolver.getOnlineMajorMapURL()
-        : resolver.getAsuCritTrackUrl();
+        : resolver.getGeneralDegreeMajorMap();
 
-      const directMapLink = (
-        <a href={directUrl}>{resolver.getRequiredCoursesLabel()} Map</a>
-      );
+      let directMapLink;
+
+      try {
+        const url = new URL(directUrl);
+        directMapLink = (
+          <a href={directUrl}>{resolver.getRequiredCoursesLabel()} Map</a>
+        );
+      } catch (error) {
+        return "";
+      }
```

Comment:
We need something like this logic implemented at https://github.com/ASU/asu-unity-stack/blob/dpl-degree-switchover/packages/app-degree-pages/src/components/ListingPage/components/ProgramList/AccordionView/index.js#L58 because the mobile and tablet accordion view includes the Required Courses major map link as an empty link for graduate. You'll probably want to include the label in that logic so it doesn't display when the link is absent.